magic-modules/merge-pr.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+echo "$CREDS" > ~/github_private_key
+chmod 400 ~/github_private_key
+
+
+shopt -s dotglob
+cd mm-approved-prs
+ID=$(git config --get pullrequest.id)
+BRANCH=$(git config --get pullrequest.branch)
+cd ..
+cp -r mm-approved-prs/* mm-output
+
+cd mm-output
+git config pullrequest.id $ID
+git checkout $BRANCH
+git config --global user.email "nmckinley@google.com"
+git config --global user.name "Magic Module Bot"
+git config -f .gitmodules submodule.build/terraform.branch master
+git config -f .gitmodules submodule.build/terraform.url "git@github.com:terraform-providers/terraform-provider-google.git"
+ssh-agent bash -c "ssh-add ~/github_private_key; git submodule update --remote --init build/terraform"
```

Comment:
If instead you make the github_private_key -> .ssh/id_rsa you can scrap the ssh-agent requirement.

Reply:
`~/.ssh/id_rsa`, right?  I tried that, but `~` is `/root/` here, and `/root/.ssh/id_rsa` didn't work when I tried it (for whatever reason), so this solution seemed workable to me.

Reply:
It is not a matter of workable. It just looks like an unnecessary layer. I'll investigate and report back.